rosdep/base.yaml

```diff
@@ -3113,7 +3113,11 @@ libturbojpeg:
   fedora: [turbojpeg-devel]
   freebsd: [libjpeg-turbo]
   gentoo: [media-libs/libjpeg-turbo]
-  ubuntu: [libturbojpeg, libjpeg-turbo8-dev]
+  ubuntu: 
```

Comment:
the `debian` rule needs to be updated as well, it currently lists `libturbojpeg1-dev` that is not available on Stretch and newer.
We should update it to use [`libturbojpeg1-dev` for jessie](https://packages.debian.org/jessie/libturbojpeg1-dev) and [`libturbojpeg0-dev` for all other distros](https://packages.debian.org/stretch/libturbojpeg0-dev)